src/app/public/modules/datepicker/date-formatter.ts

```diff
@@ -5,10 +5,10 @@ export class SkyDateFormatter {
     return moment(date.getTime()).format(format);
   }
 
-  public getDateFromString(dateString: string, format: string): Date {
+  public getDateFromString(dateString: string, format: string, strict: boolean = false): Date {
     let momentValue = moment(dateString, format);
 
-    if (!momentValue.isValid()) {
+    if (!strict && !momentValue.isValid()) {
       momentValue = moment(dateString, 'YYYY-MM-DDThh:mm:ss.sssZ');
```

Comment:
Curious why you're not passing the `strict` value into the `moment` function directly?
```
let momentValue = moment(dateString, format, strict);
```

Reply:
Apologies, I didn't have all the code checked in. I can see why you were confused! I'm now using the strict property as you suggest.